### PR TITLE
Fix service chain rpc

### PR DESCRIPTION
**Pull request by hyeonLewis**
## Proposed changes

Fix sc's mainbridge to register Kaia namespace.

This PR brings @blukat29's [hotfix](https://github.com/blukat29/klaytn/tree/hotfix/scn-to-klay-rpc)

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia/blob/main/CONTRIBUTING.md) doc
- [ ] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: networks/rpc/service.go [hunk 1]
```diff
@@ -118,19 +120,25 @@ func (r *serviceRegistry) callback(method string) *callback {
 		namespace = "kaia"
 	}
 
-	return r.services[namespace].callbacks[method]
+	cb := r.services[namespace].callbacks[method]
+	logger.Trace("lookup callback", "method", reqMethod, "ok", cb != nil)
+	return cb
```
Comment (blukat29) [inline networks/rpc/service.go:125]: @hyeonLewis No need to bring in logging in this file. I added it for debugging, but it's too wordy here.

Comment (hyeonLewis) [reply networks/rpc/service.go:125]: Ok, I thought it needs to be identical. Then can I also apply cleanup like below?
```go
func (s *Server) RegisterName(name string, rcvr interface{}) error {
	if name == "klay" || name == "kaia" {
		if err := s.services.registerName("klay", rcvr); err != nil {
			return err
		}
		if err := s.services.registerName("kaia", rcvr); err != nil {
			return err
		}
		return nil
    } 
    return s.services.registerName(name, rcvr)
}
```

Comment (hyeonLewis) [reply networks/rpc/service.go:125]: Ok, I thought it needs to be identical. Thanks.

Comment (hyeonLewis) [reply networks/rpc/service.go]: Applied and force-pushed.

**Review by blukat29 — APPROVED** [review]

**Review by 2dvorak — APPROVED** [review]
